### PR TITLE
refac: change signature of Cognite3DViewer.models

### PR DESCRIPTION
**Pull request by larsmoa**
# Description

As discussed in 4.0 change document.

# Checklist:

Not filled out, too small of a change.

### CODE REVIEW

File: viewer/packages/api/src/public/migration/Cognite3DViewer.ts [hunk 1]
```diff
@@ -187,8 +187,8 @@ export class Cognite3DViewer {
   /**
    * Gets a list of models currently added to the viewer.
    */
-  public get models(): CogniteModelBase[] {
-    return this._models.slice();
```
Comment (haakonflatval-cognite) [inline viewer/packages/api/src/public/migration/Cognite3DViewer.ts:191]: Should we just change the type of `this._models` to `(Cognite3DModel | CognitePointCloudModel)[]` instead?

**Review by haakonflatval-cognite — CHANGES_REQUESTED** [review]
See comment

**Comment by christjt** [comment]
The biggest pain point I have with CogniteModelBase is that it is a base class. We should either delete it compleatly or convert it / devide it into interfaces imo.

**Comment by larsmoa** [comment]
Ok, I see the comments - I'll just close this for now.

**Pull request closed without merging**